app/views/affiliates/index.html.haml

```diff
@@ -1,22 +1,26 @@
 = content_for :h1 do
-  Global Settings
+  = t("pages.global_settings")
 
 = content_for :sidebar do
-  = render :partial => 'admin/shared/sidenav_global', :locals => { :sidenav_tab => 'affiliates' }
+  = render "admin/shared/sidenav_global", sidenav_tab: "affiliates"
 
-%h2 Affiliates
-= link_to 'Add Affiliate', new_affiliate_path, :class => 'btn-add'
+%h2= t("pages.affiliates")
+= link_to t("affiliates.add"), new_affiliate_path, class: "btn-add"
 
 %table.table.table-striped.table-hover
   %thead
     %tr
       %th
-      %th{:width => '100%'} Affiliate
+      %th{width: "100%"}= t("affiliates.th")
   %tbody
-    - Affiliate.all.each do |affiliate|
-      - next unless affiliate.destroyable?
+    - Affiliate.all.select(&:destroyable?).each do |affiliate|
       %tr
-        %td= link_to('Remove', affiliate_path(affiliate), :confirm => "Really remove affiliate #{affiliate.name}?", :method => :delete)
+        %td
+          = link_to t("affiliates.remove.label"),
+            affiliate_path(affiliate),
+            confirm: t("affiliates.remove.confirm", name: affiliate.name),
+            method: :delete
         %td.indented_lots
           = affiliate.name
-          %span.indented== (#{link_to('Edit', edit_affiliate_path(affiliate))})
+          %span.indented
+            == (#{link_to(t("affiliates.edit"), edit_affiliate_path(affiliate))})
```

Comment:
Seems like we don't use the `==` very often. I actually had to look up what it meant. Would
`= %[#{link_to(t("affiliates.edit"), edit_affiliate_path(affiliate))}]` be more obvious (although there are already so many parentheses and curly braces, it's pretty confusing as is).